### PR TITLE
Added Copy Group to Mappings Client

### PR DESCRIPTION
**Pull request by smmr-dn**
Added new method for Copy Group in Mapping Client. Added new base path for Grouping and Mapping (https://api.bentley.com/grouping-and-mapping/)

### CODE REVIEW

File: src/reporting/clients/IMappingsClient.ts [hunk 1]
```diff
@@ -163,6 +163,20 @@ export interface IMappingsClient {
     group: GroupCreate
   ): Promise<Group>;
 
+  /**
+   * Copy a Group for a Mapping.
+   * @param {string} accessToken OAuth access token with scope `insights:modify`.
```
Comment (MindaugasButkus) [inline src/reporting/clients/IMappingsClient.ts:168]: The required scope is `imodels:modify`.

**Comment by MindaugasButkus** [comment]
Bump patch version in package.json and add entry in CHANGELOG.md.

File: src/reporting/clients/IMappingsClient.ts (hunk 1 above)
Comment (smmr-dn) [reply src/reporting/clients/IMappingsClient.ts]: I have changed it to `imodels:modify`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Change Log - @itwin/insights-client
 
+## 0.5.4
+Tuesday, 12 December, 2023
+### Minor
+- Added `GroupCreateCopy` and `SourceGroupReference` interfaces.
+- Added `GROUPING_AND_MAPPING_BASE_PATH` in `OperationsBase.ts`.
```
Comment (MindaugasButkus) [inline CHANGELOG.md:7]: Don't need to specify where you added it. Packages will be able to import it with
```
import { GROUPING_AND_MAPPING_BASE_PATH } from "@itwin/insights-client";
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+- Added `copyGroup` function in `MappingsClient.ts` to implement group copying functionaliy.
+- Added the unit test and integration test for the new group copying functionality.
```
Comment (MindaugasButkus) [inline CHANGELOG.md:9]: No need to have information about tests in release notes.

File: CHANGELOG.md (hunk 2 above)
Comment (smmr-dn) [reply CHANGELOG.md]: I have updated the changelog

File: CHANGELOG.md (hunk 3 above)
Comment (smmr-dn) [reply CHANGELOG.md]: I have updated the changelog

**Review by MindaugasButkus — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Change Log - @itwin/insights-client
 
+## 0.5.4
+Tuesday, 12 December, 2023
+### Minor
```
Comment (Guillar1) [inline CHANGELOG.md:5]: Minor change but only a patch bump?

Comment (MindaugasButkus) [reply CHANGELOG.md:5]: I always only check the message and miss the category of the change :|

Comment (MindaugasButkus) [reply CHANGELOG.md:5]: I see it's fixed in https://github.com/iTwin/insights-client/pull/48

**Pull request merged**